spec/spec_helper.rb

```diff
@@ -1,5 +1,6 @@
 require "factory_girl_rails"
 require "coveralls"
+require 'mocha'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.